addons/sale/models/sale_order_line.py

```diff
@@ -600,14 +600,16 @@ def _convert_to_tax_base_line_dict(self, **kwargs):
         :return: A python dictionary.
         """
         self.ensure_one()
+        quantity = (self.product_id.invoice_policy == 'delivery' and max(self.product_uom_qty, self.qty_delivered)
+                    or self.product_uom_qty)
```

Comment:
```suggestion
        if self.product_id.invoice_policy == 'delivery':
            quantity =  max(self.product_uom_qty, self.qty_delivered)
        else:
            quantity = self.product_uom_qty
```
RD Sales' style guide has a 100 char line length limit, plus an if-else makes it a bit easier to read